packages/grpc-js-xds/src/xds-client.ts

```diff
@@ -340,6 +340,7 @@ export class XdsClient {
         if (this.hasShutdown) {
           return;
         }
+        trace('Loaded bootstrap info: ' + JSON.stringify(bootstrapInfo, undefined, 2));
```

Comment:
Is it possible to print the final bootstrap with the build_version, user_agent_name, server_features fields?

Reply:
In this code that's not really treated as the "final bootstrap", but I can log the final `Node` messages that will be sent in the ADS and LRS requests.

Reply:
I added logging for those too.